docs/content/surfacers/overview.md

```diff
@@ -48,3 +49,66 @@ surfacer {
 }
 ```
 
+### Filtering Metrics
+
+It is possible to filter the metrics that the surfacers receive.
+
+#### Filtering by Label
+
+Cloudprober can filter the metrics that are published to surfacers. To filter metric labels, reference one of the following keys in the surfacer configuration:
```

Comment:
s/To filter metric labels/To filter metrics by labels/

Reply:
Thanks, fixed here: https://github.com/google/cloudprober/pull/598/commits/4a4c5e982badc7944488fa1ad4fe7e38e2d3a802